source/Halibut/Transport/Protocol/WebSocketStream.cs

```diff
@@ -189,7 +190,7 @@ protected override void Dispose(bool disposing)
             {
                 try
                 {
-                    SendCloseMessage();
+                    SendCloseMessage().GetAwaiter().GetResult();
```

Comment:
And change this back to calling the existing previous SendCloseMEssage

Reply:
Done